dashboard/lib/tasks/asset_sync.rake

```diff
@@ -18,10 +18,11 @@ namespace :assets do
   desc 'Copy digested assets to non-digested file paths'
   task no_digests: :environment do
     assets = Dir.glob("#{dashboard_dir}/public/assets/js/**/*")
-    regex = /-\w{32}(?:\.\w+$)/
+    regex = /(^.*)-\w{32,64}(\.\w+$)/
     assets.each do |file|
       next if File.directory?(file) || file !~ regex
-      non_digested = file.sub(regex, '')
+      non_digested = file.gsub(regex, '\1\2')
+      puts "Copying file #{file} to #{non_digested}"
```

Comment:
Agree with a comment of Brendan's on an earlier PR that it might be useful to have a comment with an example of a before file and an after file. Also maybe an explanation of why we need to copy digested assets to non-digested file paths?